### PR TITLE
[docker] Install Verilator and OpenOCD from OBS

### PR DESCRIPTION
**Pull request by rswarbrick**
This matches what we do in CI and should be both more stable and also
a bit quicker.

### CODE REVIEW

File: util/container/Dockerfile [hunk 1]
```diff
@@ -78,7 +61,7 @@ RUN apt-get update && \
         libtool \
         minicom \
         screen && \
-    apt-get clean ; \
+    apt-get clean && \
```
Comment (imphil) [inline util/container/Dockerfile:64]: That is actually intentional (not that it matters that much)

**Review by imphil — APPROVED** [review]
Lovely (TM). Thanks!

File: util/container/Dockerfile (hunk 1 above)
Comment (rswarbrick) [reply util/container/Dockerfile:64]: Oh, good point. I'll put it back. Also (reading the Dockerfiles best practices page), I think there's a missing `apt-get update` which I'll add in a sec.

**Comment by rswarbrick** [comment]
Merging this now: the docker part of CI has passed and this doesn't touch anything else.

**Comment by imphil** [comment]
Nice, this brings down the time to build the docker image from 12 to 5 minutes (it's not in the critical path, but still reduces the amount of resources we use. A win for the planet.)

**Comment by rswarbrick** [comment]
2 days before Earth day!

**Pull request merged**